.github/workflows/process_test.yml

```diff
@@ -0,0 +1,45 @@
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, edited]
+  workflow_dispatch:
+  
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"        
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          path: "main"
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Install dependencies
+        shell: bash
+        run: |
+          python -m pip install --upgrade pip
+          pip install wheel
+          cd $GITHUB_WORKSPACE/main
+          pip install .  
+      - name: Clone AWS Level 0 data repo for testing
+        env:
+          GITLAB_TOKEN : ${{ secrets.GITLAB_TOKEN }}
+        run: | 
+          cd $GITHUB_WORKSPACE
+          git clone --depth 1 https://oauth2:${{ env.GITLAB_TOKEN }}@geusgitlab.geus.dk/glaciology-and-climate/promice/aws-l0.git
+      - name: Run data processing
+        env:
+          TEST_STATION: KPC_U CEN2 JAR
+        shell: bash
+        run: |
+          mkdir $GITHUB_WORKSPACE/out/
+          for i in $(echo ${{ env.TEST_STATION }} | tr ' ' '\n'); do
+            python3 $GITHUB_WORKSPACE/main/bin/getL3 -v $GITHUB_WORKSPACE/main/src/pypromice/variables.csv -m $GITHUB_WORKSPACE/main/src/pypromice/metadata.csv -c $GITHUB_WORKSPACE/aws-l0/raw/config/$i.toml -i $GITHUB_WORKSPACE/aws-l0/raw -o $GITHUB_WORKSPACE/out/
```

Comment:
So, this is the "meat" of what the github action actually runs. I am curious how we are actually testing the inputs/outputs? Are we just making sure everything runs without any uncaught errors? Is there any checking of the content or structure of outputs?

Reply:
We don't test the inputs. This is mainly to make sure everything runs. The output is provided as a Github Action artifact that can be viewed after a successful run. 

We could implement something after this line that checks the output so that we don't need to view and check it ourselves.